clang/tools/scan-build-py/lib/libear/ear.c

```diff
@@ -492,35 +493,14 @@ int eaccess(const char *pathname, int mode) {
       pathname[1] == 'v' && pathname[0] == 'n') {
     // To handle case like "nvcc foo.cu ..."
     return 0;
-  } else if (len > 4 && pathname[len - 1] == 'c' && pathname[len - 2] == 'c' &&
-             pathname[len - 3] == 'v' && pathname[len - 4] == 'n' &&
-             pathname[len - 5] == '/') {
-    // To handle case like "/path/to/nvcc foo.cu ..."
-    return 0;
   }
-  return call_eaccess(pathname, mode);
-}
-
```

Comment:
I think we should keep this.

Reply:
Yes, we keep it, just do a little refinement.